javascript/UserForm.js

```diff
@@ -109,12 +124,21 @@ jQuery(function ($) {
 		},
 		// When a field becomes valid.
 		success: function (error) {
-			var errorId = $(error).attr('id');
+			var errorId = $(error).attr('id'),
+				fieldId = errorId.substr(0, errorId.indexOf('-error')),
```

Comment:
Does this mean errorID is in the form `id="field[]-error"`? In which case this is the problem here.